prebuilt/core/booking.json

```diff
@@ -699,6 +699,11 @@
               "required": [
                 "classification"
               ]
+            },
+            "startEndGeoRegionUrl": {
```

Comment:
Can we just define this in one place, and [reference](https://cswr.github.io/JsonSchema/spec/definitions_references/#references) it where it is needed? I.e. there is a lot of redundancy here.

Reply:
It is done that way. This is the commit that exposes whole change: https://github.com/maasglobal/maas-schemas/pull/221/commits/760fd0b326cbdd4171018cf76406cc3813293375

Still, we also version schema files with all references resolved (stored in `prebuilt` folder), whether that's needed, is another subject for discussion, but not in scope of this PR, and that's why you see it many places. Result of build step was committed with separate commit: https://github.com/maasglobal/maas-schemas/pull/221/commits/7375d82a4c8b6d0a173922693d5b9dc1ec5a1a09

Reply:
Ah, I didn't know we have a prebuilt version. That makes sense now.